Src/IronPython/Compiler/Ast/FunctionDefinition.cs

```diff
@@ -626,18 +626,6 @@ private class FunctionDefinitionInstruction : Instruction {
 
             CreateFunctionVariables(locals, init);
 
-            // If the __class__ variable is used in a class method then we need to initialize it.
-            // This must be done before parameter initialization (in case one of the parameters is called __class__).
-            ClassDefinition parent = FindParentOfType<ClassDefinition>();
```

Comment:
I think this was the last call to `FindParentOfType` so maybe we can remove the method?

Reply:
Good catch